jax/_src/numpy/lax_numpy.py

```diff
@@ -2910,11 +2910,14 @@ def nanmax(a, axis: Optional[Union[int, Tuple[int, ...]]] = None, out=None,
 @_wraps(np.nansum, skip_params=['out'])
 @partial(jit, static_argnames=('axis', 'dtype', 'keepdims'))
 def nansum(a, axis: Optional[Union[int, Tuple[int, ...]]] = None, dtype=None,
-           out=None, keepdims=None):
+           out=None, keepdims=None, initial=None, where=None):
   lax._check_user_dtype_supported(dtype, "nanprod")
   return _nan_reduction(a, 'nansum', sum, 0, nan_if_all_nan=False,
                         axis=axis, dtype=dtype, out=out, keepdims=keepdims)
 
+# Work around a sphinx documentation warning in NumPy 1.22.
+nansum.__doc__ = nansum.__doc__.replace("\n\n\n", "\n\n")
```

Comment:
Maybe better to do this as part of `_wraps`?

Reply:
I think it's a documentation bug in this particular docstring. The right fix is probably to send a PR upstream to remove the extra newline..